libcxx/test/std/language.support/support.exception/uncaught/uncaught_exception.pass.cpp

```diff
@@ -7,6 +7,10 @@
 //===----------------------------------------------------------------------===//
 
 // UNSUPPORTED: no-exceptions
+
+// ADDITIONAL_COMPILE_FLAGS: -D_LIBCPP_ENABLE_CXX20_REMOVED_UNCAUGHT_EXCEPTION
```

Comment:
Can you add a new `.verify` test to validate the deprecation?
look at `test/std/utilities/function.objects/negators/unary_negate.depr_in_cxx17.verify.cpp` for an example.

Reply:
Added `test/std/language.support/support.exception/uncaught/uncaught_exeption.depr_in_cxx17.verify.cpp`.